Make.config

```diff
@@ -30,28 +31,31 @@ PACKAGE_HEAD_REV=$(shell git rev-parse HEAD)
 # on wrench, because wrench technically builds hashes, not branches)
 #
 # 
-PACKAGE_HEAD_BRANCH=master
+PACKAGE_HEAD_BRANCH=xcode9
 ifeq ($(BUILD_REVISION),)
 CURRENT_BRANCH:=$(shell git rev-parse --abbrev-ref HEAD)
 else
 CURRENT_BRANCH:=$(PACKAGE_HEAD_BRANCH)
 endif
 
+# for service releases and previews
+PACKAGE_VERSION_REV=0
+
 IOS_PRODUCT=Xamarin.iOS
 IOS_PACKAGE_NAME=Xamarin.iOS
 IOS_PACKAGE_NAME_LOWER=$(shell echo $(IOS_PACKAGE_NAME) | tr "[:upper:]" "[:lower:]")
 # NEVER customize IOS_PACKAGE_VERSION itself, other parts (mtouch, web updater) are using the IOS_PACKAGE_VERSION_* variables
-IOS_PACKAGE_VERSION=10.15.0.$(IOS_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=11.5.$(PACKAGE_VERSION_REV).$(IOS_COMMIT_DISTANCE)
```

Comment:
11.5 doesn’t look right 😃

Reply:
@rolfbjarne the final will like @VincentDondain mentioned
but yes I skipped a step  ;-) since `d15-5` (11.4) will be branched after the merge